### ISSUE TITLE
Timelines not visible after startup

### ISSUE THREAD

**Issue by veskuh** [veskuh/Tweetian#185]
Some users on Vaarainjärvi have reported timelines not becoming visible and Tweetian being unresponsive on startup

**Comment by anandrkris** [comment]
Yes, can confirm that faced this issue with the official version. Openrepos version works fine though.

**Comment by nomeata** [comment]
Is the timeline the second panel? Always has been empty here (using the official version).

**Comment by nomeata** [comment]
And indeed, the openrepo version (2.0.6) works.
